src/components/review-cards.jsx

```diff
@@ -1,8 +1,106 @@
 import React from 'react';
 
 class ReviewCards extends React.Component{
+    constructor(props){
+        super(props);
+
+    this.state={
+        isToggled: false,
+    }
+
+    this.handleToggle = this.handleToggle.bind(this);
+    this.nextCard = this.nextCard.bind(this);
+    this.displayFrontBack = this.displayFrontBack.bind(this);
+    this.previousCard = this.previousCard.bind(this);
+   
+    }
+
+    componentDidMount(){
+        if(typeof this.props.cards[0] !== 'undefined'){
+            this.props.setActiveCard(0);
+        }  
+    }
+
+    handleToggle(){
+        return this.setState({
+            isToggled: !this.state.isToggled
+        });
+    }
+
+    displayFrontBack(){
+        
+        if(this.state.isToggled){
+            return <div className="card-body">
+                        <h5 className="card-title">Answer:</h5>
+                        <p className="card-text">{this.props.activeCard ? this.props.activeCard.answer : ''}</p>
+                    </div>
+        }
+        else{
+            return <div className="card-body">
+                        <h5 className="card-title">Question:</h5>
+                        <p className="card-text">{this.props.activeCard ? this.props.activeCard.question : ''}</p>
+                    </div>
+        }
+
+    }
+
+    
+    nextCard(){
+        this.state.isToggled = false;
```

Comment:
Although it probably seemed like everything was working you never want to directly set a value to `state`. This will cause undesired side effects because React won't be aware of the change. Whenever changing a value in `state` always use `setState`.

```JAVASCRIPT
this.setState({
  isToggled: false
});
```